### PR TITLE
Fix tier4 OSCAP test

### PR DESCRIPTION
**Pull request by oshtaier**
Waiting for https://github.com/SatelliteQE/robottelo/issues/3698 to be implemented
```
nosetests tests/foreman/longrun/test_oscap.py
.
----------------------------------------------------------------------
Ran 1 test in 771.578s

OK
```

### CODE REVIEW

File: tests/foreman/longrun/test_oscap.py [hunk 1]
```diff
@@ -262,7 +265,6 @@ def test_positive_upload_to_satellite(self):
                     ssh.command(u'smart-proxy-openscap-send')
```
Comment (kbidarkar) [inline tests/foreman/longrun/test_oscap.py]: This command is no longer required for Sat6.2.

Comment (oshtaier) [inline tests/foreman/longrun/test_oscap.py]: Right, working fine without it

**Comment by oshtaier** [comment]
```
 nosetests tests/foreman/longrun/test_oscap.py
.
----------------------------------------------------------------------
Ran 1 test in 774.579s

OK
```

**Comment by rochacbruno** [comment]
ACK 
(once #3709 is merged the next ack can merge this)

**Comment by sghai** [comment]
ACK..

**Pull request merged**